docs/ops/doc/CallingOps.md

```diff
@@ -1,61 +1,96 @@
 # Calling Ops with the `OpBuilder`
 
-Ops are designed to be called from the Op matcher, using the `OpBuilder` syntax. OpBuilder chains follow the [builder pattern](https://refactoring.guru/design-patterns/builder), allowing users to create complex Op calls by "chaining" or appending consecutive, simpler method calls.
+Use of the Ops framework centers on a process of matching Op requests to algorithm implementations based on the parameters provided. The easiest way to make these queries is to use the `OpBuilder` syntax, which follows the [builder pattern](https://refactoring.guru/design-patterns/builder) to assemble the required components of an Op matching request from a particular `OpEnvironment`.
 
-On this page, we will be constructing an `OpBuilder` call on an `OpEnvironment ops` to execute a Gaussian Blur on an input image `inImage`, with our output buffer `outImage`.
+In this page, we start after having [identified a Gaussian Blur Op](SearchingForOps) that we would like to use. We assume we already have created an `OpEnvironment` named `ops`, as well as the input image to blur, and a pre-allocated output image for the result&mdash;`inImage` and `outImage`, respectively.
+
+**Note:** we are incrementally constructing one line of code in this example. Running an intermediate step simply returns an appropriate builder that knows what has been set so far, and which step is next. If you're following along in an IDE or script editor, the code you actually *run* would be the last step, once our builder call is fully constructed.
 
 ## Specifying the name with `.op()`
 
-From the `OpEnvironment`, an `OpBuilder` call is initialized with the method `OpEnvironment.op(String)`, which is used to describe the name of the Op that the `OpBuilder` must return:
+From the `OpEnvironment`, an `OpBuilder` chain is initialized with the `op(String)` method, which describes the name of the Op that we ultimately want to call:
 
 ```groovy
 ops.op("filter.gauss")
 ```
 
 ## Passing the inputs with `.input()`
 
-With the name defined in the `OpBuilder` call, we can then chain the inputs with the `input()` method.
+With the name established in the `OpBuilder` chain, we can then specify our input(s) with the `.input()` method.
 
-For our gaussian blur, we will pass as inputs our input image `inImage`, and a `double` as our sigma parameter:
+For this Gaussian blur, we have two inputs: `inImage` is the image we want to blur, and a `double` value as our sigma parameter:
```

Comment:
```suggestion
For this Gaussian blur, we have two inputs: `inImage` is the image we want to blur, and a `double` value as the standard deviation of the smoothing function:
```

What do you think about using the technical term?

Reply:
I like the simpler `our sigma parameter` :wink: